shards/shards_test.go

```diff
@@ -513,76 +513,82 @@ func BenchmarkShardedSearch(b *testing.B) {
 	}
 }
 
-func TestVisibilityQuery(t *testing.T) {
+func TestRawQuerySearch(t *testing.T) {
 	ss := newShardedSearcher(1)
 
 	var nextShardNum int
-	addShard := func(repo string, public bool, docs ...zoekt.Document) {
+	addShard := func(repo string, rawConfig map[string]string, docs ...zoekt.Document) {
 		r := &zoekt.Repository{Name: repo}
-		if public {
-			r.RawConfig = map[string]string{"public": "1"}
-		}
+		r.RawConfigEncoded = zoekt.EncodeRawConfig(rawConfig)
```

Comment:
this is somewhat of a sign to me that this is wrong? IE we shouldn't need to specify both RawConfigEncoded and RawConfig when creating a Repository. It should be created in the indexData from Repository.

Reply:
done